integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java

```diff
@@ -28,9 +28,7 @@
 import org.junit.jupiter.api.DisplayName;
```

Comment:
Modify the Copyright statement to include year 2023

Reply:
Changed